lib/Constants/links.dart

```diff
@@ -0,0 +1,4 @@
+class Links {
```

Comment:
@adizcode I don't think we need a separate file to store a single path. You can directly add it as a string where you are using the URLLauncher.